modules/v2x/BUILD

```diff
@@ -1,4 +1,3 @@
-load("@rules_cc//cc:defs.bzl", "cc_binary")
```

Comment:
Why this line got removed?

Reply:
Removed by ```apollo.sh format```

Reply:
Oops, It seems you found a bug in `apollo.sh format`. 

If you run `scripts/buildifier.sh path/to/this/BUILD`, the `load` statement should be back.
It doesn't matter. We can fix it in later PRs.